lib/novu/http.ex

```diff
@@ -91,7 +110,8 @@ defmodule Novu.Http do
     [
       {"accept", "application/json"},
       {"content-type", "application/json"},
-      {"authorization", "ApiKey #{api_key()}"}
+      {"authorization", "ApiKey #{api_key()}"},
+      {"Idempotency-Key", UUID.uuid4()}
```

Comment:
Do we need the case sensitivity here?

```suggestion
      {"idempotency-key", UUID.uuid4()}
```

Reply:
No, it's unnecessary